services/web/server/src/simcore_service_webserver/diagnostics_entrypoints.py

```diff
@@ -49,19 +52,48 @@ async def get_diagnostics(request: web.Request):
     return web.json_response(data)
 
 
+async def get_app_status(request: Request):
+    # TODO: add tester required
+    check = AppStatusCheck.parse_obj(
+        {
+            "name": app_name,
+            "version": api_version,
+            "services": {
+                "postgres": {
+                    "healthy": await db.is_service_responsive(request.app),
+                    "pool": db.get_engine_state(request.app),
+                },
+                "storage": {"healthy": await storage_api.is_healthy(request.app)},
+                "director_v2": {"healthy": await director_v2.is_healthy(request.app)},
```

Comment:
Also make sure this reply fast or timeout quickly. The result of this call should be fast.

Reply:
health-checks are now concurrent